test/bdd/pkg/vc/vc_steps.go

```diff
@@ -329,14 +329,14 @@ func (e *Steps) retrieveCredential(profileName string) error {
 		return expectedStatusCodeError(http.StatusOK, resp.StatusCode, respBytes)
 	}
 
-	unescapedResponse, err := strconv.Unquote(string(respBytes))
-	if err != nil {
-		return err
-	}
-
 	expectedCredential := string(e.bddContext.CreatedCredential)
-	if unescapedResponse != expectedCredential {
-		return expectedStringError(expectedCredential, unescapedResponse)
+	receivedCredential := string(respBytes)
+	// For some reason there's an extra line at the end of the credential returned from the create credential handler.
+	// By adding a space to the received credential we just received we should be able to directly compare the strings.
+	receivedCredential += `
```

Comment:
??

Reply:
@fqutishat To clarify: the extra line that I noticed is unrelated to my change, so I created a followup story to investigate and fix it in order to avoid scope creep here.

https://github.com/trustbloc/edge-service/issues/104

Note however that the extra line may be coming from Aries-Framework-Go, since we call into that code while creating a credential. So I (or someone else) might have to make a fix there first.

In the meantime, I've replaced this with the JSON trimming suggestion per @Baha-sk.